### ISSUE TITLE
Change salary as a Boss/Owner of a job

### ISSUE THREAD

**Issue by Birdy113** [Qbox-project/qbx_core#235]
### The problem

You cannot change salary as boss/owner.  From old days when old ESX we could do that but not with newer frameworks. I miss this feauture 

### Ideal solution

Give freedom to business owners (Boss) to change salary to their employees to create better business RP

### Alternative solutions

Make it in DB salary system or make it possible to do salary in DB. (Changeable)

**Comment by Manason** [comment]
Yeah this is something I'd love to do in the future when I have some time. The way we'd do it is adding a separate jobs table which lists all the job names and their salaries. Then the existing jobs config would just be the default salary if there is no value in the database.

If someone can get to it before me though that would be great.

**Comment by TonybynMp4** [comment]
> Yeah this is something I'd love to do in the future when I have some time. The way we'd do it is adding a separate jobs table which lists all the job names and their salaries. Then the existing jobs config would just be the default salary if there is no value in the database.
> 
> If someone can get to it before me though that would be great.

might be something we could do with 2.0 and the great db refactor / new group system

**Comment by Manason** [comment]
> > Yeah this is something I'd love to do in the future when I have some time. The way we'd do it is adding a separate jobs table which lists all the job names and their salaries. Then the existing jobs config would just be the default salary if there is no value in the database.
> > If someone can get to it before me though that would be great.
> 
> might be something we could do with 2.0 and the great db refactor / new group system

No need to wait for 2.0. This would be a separate new table. Also has nothing to do with a group system.

**Comment by Manason** [comment]
Attempted to do this, but it was too difficult to maintain backwards compatibility. To do this properly, we'd need to likely remove the jobs.lua entirely and create a set of commands and a UI for modifying the jobs. And we'd need to break QBCore.Shared.Jobs. Will mark this as a v2.0 change
